pkg/manifest/manifest.go

```diff
@@ -256,23 +257,33 @@ func (manifest *manifest) RunAsUser() string {
 }
 
 func (manifest *manifest) UnpackAsUser() string {
-	if manifest.ReadOnly {
```

Comment:
you'll have to check that `manifest.ReadOnly` isn't nil before dereferencing it otherwise it'll panic

Reply:
you probably just want `manifest.GetReadOnly()` here?

Reply:
Fixed